lib/views/help/unhappy.html.erb

```diff
@@ -111,7 +111,7 @@
 
   <p>
     To make a referral,
-    start by reading <a href="https://ico.org.uk/make-a-complaint/foi-and-eir-complaints/">the
```

Comment:
This URL (now?) works, but the addition saves a redirect 👍

Reply:
Looks like they fixed it - the page was doing something funky until recently! 😅